pipelines/SpeciesTreeFromBusco/nextflow.config

```diff
@@ -79,22 +83,17 @@ params {
     gffread_exe = "$COMPARA_SOFTWARE/shared/build/gffread/0.12.7/bin/gffread"
     // Path to miniprot binary:
     miniprot_exe = "$COMPARA_SOFTWARE/shared/build/miniprot/0.12/bin/miniprot"
-    // Path to mafft binary:
-    mafft_exe = "$LINUXBREW_HOME/bin/mafft"
     // Path to iqtree2 binary:
     iqtree_exe = "$COMPARA_SOFTWARE/shared/build/iqtree2/2.2.0.3/bin/iqtree2"
-    // Path to gotree binary:
-    gotree_exe = "$COMPARA_SOFTWARE/shared/build/gotree/0.4.4a/bin/gotree"
     // Path to trimal binary:
     trimal_exe = "$LINUXBREW_HOME/bin/trimal"
     // Path to seqkit binary:
     seqkit_exe = "$COMPARA_SOFTWARE/shared/build/seqkit/2.2.0/bin/seqkit"
-    // Path to pal2nal script:
-    pal2nal_exe = "$COMPARA_SOFTWARE/shared/build/pal2nal/14/bin/pal2nal"
-    run_pal2nal_exe = "$ENSEMBL_ROOT_DIR/ensembl-compara/pipelines/SpeciesTreeFromBusco/scripts/run_pal2nal.pl"
     // Path to astral JAR:
     astral_jar = "$COMPARA_SOFTWARE/shared/build/astral/5.7.1/Astral/astral.5.7.1.jar"
-    // Path to MACSE JAR:
-    macse_jar = "$COMPARA_SOFTWARE/shared/build/macse/2.06/macse_v2.06.jar"
+    // Path to pagan:
+    pagan_exe = "/homes/sbotond/bin/pagan_bin/bin/pagan"
```

Comment:
Perhaps `pagan_exe` should go somewhere under `$COMPARA_SOFTWARE`?

Reply:
I agree!  How about:
  pagan_exe = "$COMPARA_SOFTWARE/shared/build/pagan/pagan_version/pagan"

Reply:
Fair enough! Will do a push for this.

Reply:
Done.